### PR TITLE
Migrate to GitHub Actions.

### PR DESCRIPTION
**Pull request by jezdez**
Travis CI has a new pricing model which places limits on open source.

* https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing
* https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works

Many projects are moving to GitHub Actions instead, including Jazzband projects:

* https://github.com/orgs/jazzband/projects/1
* https://github.com/jazzband-roadies/help/issues/206

This is based on https://github.com/jazzband/contextlib2/pull/26.

It drops support for Python 3.5 because GitHub Actions doesn't fully support them. It's EOL anyway.

Python 3.9 support is also added.

This also relates to #343 in that it enables the release workflow mentioned there with @dokterbob as proejct lead. The version is deducted from the last git tag automatically using [setuptools_scm](https://pypi.org/project/setuptools-scm/).

TODO:

* [x] @jezdez to add `JAZZBAND_RELEASE_KEY` to the repo secrets.

### CODE REVIEW

**Comment by jezdez** [comment]
@claudep Hey, I've added you as well since you've done some work on this library and I want to make sure we move this along. This is fully in line what all the other Jazzband projects have done FWIW.

**Comment by claudep** [comment]
Sorry, I'm no longer active on django-newsletter.

**Comment by jezdez** [comment]
> Sorry, I'm no longer active on django-newsletter.

Got it, thank you!

**Comment by jezdez** [comment]
@aleksihakli Apologies for pulling you into this, I want to move ahead with this pull request and your review would be much appreciated!

**Comment by aleksihakli** [comment]
Hey @jezdez! I've been a bit busy but I'll review this today. Feel free to ping me via mail or on GitHub in case you need any help, I'll happily help if I have the bandwidth :)

**Comment by jezdez** [comment]
> Hey @jezdez! I've been a bit busy but I'll review this today. Feel free to ping me via mail or on GitHub in case you need any help, I'll happily assist if I have the bandwidth :)

Understood and no problem, thank you for getting back to me :)

File: setup.py [hunk 1]
```diff
@@ -66,12 +63,10 @@
         'License :: OSI Approved :: GNU Affero General Public License v3',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
```
Comment (aleksihakli) [inline setup.py:69]: Should Python 3.9 also be included in the compatibility and test matrix?

Comment (aleksihakli) [inline setup.py:69]: 3.9 should also be added here, right?

**Review by aleksihakli — APPROVED** [review]
LGTM! `setup.py` package classifiers should probably also include Python 3.9 and possibly Django version classifiers?

File: setup.py (hunk 1 above)
Comment (jezdez) [reply setup.py:69]:
```suggestion
        'Programming Language :: Python :: 3.9',
```

**Pull request merged**